lua/nvim-treesitter/install.lua

```diff
@@ -103,6 +105,11 @@ local function outdated_parsers()
   end, info.installed_parsers())
 end
 
+local function is_lockfile_abi_matching(lang)
+  load_lockfile()
+  return lockfile[lang].abi <= vim.treesitter.language_version
```

Comment:
so the minimum supported version is currently not exposed in Neovim? We should create an issue and revisit this later when it's exposed or do it via FFI

Reply:
Just did a PR to do so, so this should land soon, and we will be able to add that to here